src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+
+}
+
+.logo__img {
+  padding: 10px 0;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  display: block;
+
+}
+
+.nav__list {
+  margin: 0;
+  list-style: none;
+  padding: 0;
+  display: flex;
+
+}
+
+.nav__link {
+  line-height: 60px;
+  height: 60px;
+  display: block;
+  text-decoration: none;
+  color: #000;
+  white-space: nowrap;
```

Comment:
1. Not sure why you need it
2. `position: relative;` here is also redundant

Reply:
If I remove:
Display block - than the blue undeline jumps straight under the "Äpple" text, whilst I need the line to be in the bottom of the element

Text Decoration - removes the undelines under the links

Color: is not inherited by the parent elements (for the links)

White-space - does not let the long names to wrap - as it affects when the screen size is reduced

Position relative - that was just used for testing so I am removing it =)

Reply:
oh, I meant only `white-space` property, this is a little overkill here, but well done for knowing the reason for usage